_topics/en/taproot.md

```diff
@@ -197,6 +197,12 @@ optech_mentions:
   - title: "BTCPay server #2830 adds support for P2TR receiving and spending"
     url: /en/newsletters/2021/09/15/#btcpay-server-2830
 
+  - title: "Specter v1.6.0 adds support for single-key taproot"
+    url: /en/newsletters/2021/09/22/#impervious-releases-ln-p2p-data-api
```

Comment:
Should be http://localhost:4000/en/newsletters/2021/09/22/#specter-v1-6-0-supports-single-key-taproot

Reply:
On it!

Reply:
Pushed a commit for this and squashed